cmd/skopeo/inspect.go

```diff
@@ -187,10 +201,31 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 			logrus.Warnf("Registry disallows tag list retrieval; skipping")
 		}
 	}
-	out, err := json.MarshalIndent(outputData, "", "    ")
-	if err != nil {
+	if len(opts.format) == 0 {
+		out, err := json.MarshalIndent(outputData, "", "    ")
+		if err == nil {
+			fmt.Fprintf(stdout, "%s\n", string(out))
+		}
 		return err
 	}
-	fmt.Fprintf(stdout, "%s\n", string(out))
-	return nil
+	return outputFormat(outputData, opts.format)
+}
+
+func outputFormat(data interface{}, format string) error {
+	var out formats.Writer
+	if format == "json" || format == "" { // "" for backwards compat
+		out = formats.JSONStruct{Output: data}
+	} else {
+		out = formats.StdoutTemplate{Output: data, Template: inspectFormat(format)}
+	}
+	return out.Out()
+}
+
+func inspectFormat(row string) string {
+	r := strings.NewReplacer(
+		"{{.Id}}", formats.IDString,
+		".Src", ".Source",
+		".Dst", ".Destination",
```

Comment:
What does all of this do? It doesn’t match any of the fields actually present in the data.

Reply:
Removed